### PR TITLE
Made the config errors more human-friendly.

### PR DESCRIPTION
**Pull request by matejak**

### CODE REVIEW

File: bin/oscapd-evaluate [hunk 1]
```diff
@@ -566,8 +566,10 @@ def main():
     config.prepare_dirs(cleanup_allowed=False)
     try:
         config.sanity_check()
-    except:
-        logging.exception("Configuration file failed sanity checking!")
+    except Exception as exc:
```
Comment (jan-cerny) [inline bin/oscapd-evaluate]: Can it catch a specific class of exception instead?

Comment (matejak) [reply bin/oscapd-evaluate]: The sanity checking code is throwing `RuntimeError`, but I think that we want to catch all exceptions and report back that they occurred during config file sanity checking.

**Comment by matejak** [comment]
I have tested the code with `atomic scan` and it works. Note that the `--verbose` option is mandatory to see the error message, because `atomic` hides stdout and stderr by default.

**Comment by jan-cerny** [comment]
@matejak Beautiful! I like it very much.

**Pull request merged**